src/utils/ContentContainerUtils.ts

```diff
@@ -0,0 +1,92 @@
+import { ViewStyle } from "react-native";
+import { Dimension } from "recyclerlistview";
+
+import { ContentStyle } from "../FlashListProps";
+
+export interface ContentStyleExplicit {
```

Comment:
What do you think of naming this `InternalContentStyle`? I understand where `explicit` is coming from but sounds odd. Ok to keep as-is though.